## ts-conf.diff by Neil Williams <codehelp@debian.org>
##
## use the linux input layer event interface

--- tslib-1.0.orig/etc/ts.conf
+++ tslib-1.0/etc/ts.conf
@@ -1,5 +1,5 @@
 # Uncomment if you wish to use the linux input layer event interface
-# module_raw input
+module_raw input
 
 # Uncomment if you're using a Sharp Zaurus SL-5500/SL-5000d
 # module_raw collie
